recipe/meta.yaml

```diff
@@ -12,20 +12,19 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<38]
```

Comment:
maybe add a comment why that is

Reply:
These statements are very common (in pretty much all feedstocks) and it's almost always the same reason: upstream doesn't support the skipped python versions.

Reply:
Indeed, see https://inspector.pypi.io/project/inquirer/3.1.4/packages/3c/b7/2b4a352659d78386e0f7f5405486aa1c41664666d33e9e8506640f702f70/inquirer-3.1.4-py3-none-any.whl/inquirer-3.1.4.dist-info/METADATA#line.9.